omigami/plotting/molecule_plotter.py

```diff
@@ -1,32 +1,17 @@
-from json import JSONDecodeError
-from typing import List, Tuple, Dict
-
-from PIL.PngImagePlugin import PngImageFile
-import pandas as pd
-
-try:
-    from rdkit import Chem
-    from rdkit.Chem import Draw
-    from rdkit.Chem.rdchem import Mol
-except ModuleNotFoundError:
-    import warnings
-
-    warnings.warn(
-        "You are missing the rdkit module. "
-        "Please go to Omigami's README for instructions on how to install it."
-    )
```

Comment:
why removed this warning?

Reply:
I think omigami.plotting should raise on init if the user didn't install omigami[plotting]

Reply:
I was getting this warning every time even with rdkit installed. I will put it back together after testing a bit to see if there is a better place to put it